src/main/java/teammates/common/util/Templates.java

```diff
@@ -62,8 +62,10 @@ public static String populateTemplate(String template, String... keyValuePairs)
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSessionUnpublished.html");
         public static final String FRAGMENT_SINGLE_FEEDBACK_SESSION_LINKS =
                 FileHelper.readResourceFile("userEmailTemplateFragment-feedbackSessionResendAllLinks.html");
-        public static final String USER_FEEDBACK_SESSION_RESEND_ALL_LINKS =
-                FileHelper.readResourceFile("userEmailTemplate-feedbackSessionResendAllLinks.html");
+        public static final String USER_EMAIL_UPDATE_RESEND_ALL_COURSE_LINKS =
```

Comment:
Actually, I am not sure why the template name is changing. The old naming is more clear?

Reply:
Since both of these templates are used by the same method in `EmailGenerator` for resending all course links, I thought that it would be better to name it as `reason + _RESEND_ALL_COURSE_LINKS` to make it more specific.

I'll proceed to revert it

Reply:
But your reason is not `USER_EMAIL_UPDATE` but `USER_REGKEY_REGENERATION`?

Reply:
Yeah, I named mine **`REGENERATE_STUDENT_KEY`**`_RESEND_ALL_COURSE_LINKS` (or can change to `USER_REGKEY_REGENERATION`, as you suggested).

I renamed `USER_FEEDBACK_SESSION_RESEND_ALL_LINKS` to **`USER_EMAIL_UPDATE`**`_RESEND_ALL_COURSE_LINKS`

Reply:
But the reason for `USER_FEEDBACK_SESSION_RESEND_ALL_LINKS ` is not `EMAIL_UPDATE` @junming403 ?